e2e/.editorconfig

```diff
@@ -0,0 +1,19 @@
+# http://editorconfig.org
```

Comment:
Should probably remove this file. We may want an editorconfig at some point but it will be in the root of the repository and we would need to remove all the options to ensure they comply with our other linting tools.